programs/sbf/c/src/invoked/invoked.c

```diff
@@ -45,7 +46,7 @@ extern uint64_t entrypoint(const uint8_t *input) {
     sol_assert(accounts[ARGUMENT_INDEX].data_len == 100);
     sol_assert(accounts[ARGUMENT_INDEX].is_signer);
     sol_assert(accounts[ARGUMENT_INDEX].is_writable);
-    sol_assert(accounts[ARGUMENT_INDEX].rent_epoch == 0);
+    sol_assert(accounts[ARGUMENT_INDEX].rent_epoch == ULLONG_MAX);
```

Comment:
@jackcmay I'm trying to use constants for u64::MAX. I've tried limits.h and <climits>. Neither works so far. Do you know the right way to do this?

Reply:
I ended up just using a constant in the file.